server/controllers/finance/reports/account_statement/index.js

```diff
@@ -37,15 +37,22 @@ async function report(req, res, next) {
     const rows = await generalLedger.findTransactions(options);
 
     const aggregateSql = `
-      SELECT SUM(debit_equiv) AS debit_equiv, SUM(credit_equiv) AS credit_equiv,
-        SUM(debit_equiv - credit_equiv) AS balance
+      SELECT SUM(debit_equiv) AS debit_equiv, SUM(credit_equiv) AS credit_equiv
       FROM general_ledger
-      WHERE uuid IN (?);
+      WHERE uuid = ?;
```

Comment:
This should be reverted to how it was originally.  It sums the `debit_equiv` and `credit_equiv` of all transactions in the report, so it needs to use all UUIDs.  Here, it only considers one UUID.  You can see the incorrect result below:

<img width="1166" alt="image" src="https://github.com/IMA-WorldHealth/bhima/assets/896472/e0d4c110-eeaf-4b65-861b-a77f7222f7b3">


The red should have the result of each item in the debit column and the blue should have the result of everything in the credit column.

Reply:
I had made a mistake with little of the data that was in bhima_test.